web/modules/custom/dpl_loans/src/Plugin/Block/LoanListBlock.php

```diff
@@ -75,7 +75,7 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function getThresholdConfig(): string {
     $generalSettings = $this->configFactory->get('dpl_library_agency.general_settings');
-    return $generalSettings->get('threshold_config');
+    return $generalSettings->get('threshold_config') ?? "{ 'colorThresholds': { 'danger': '0', 'warning': '6' } }";
```

Comment:
I am not a fan of hardcodet config if it is not centralized.
It should be possible to store as exported config though.

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/303/commits/dd82a896b9160af77207107dd55da9ec69c14dbb

Reply:
wie so?